tests/ReferenceSqlTest.php

```diff
@@ -535,8 +535,8 @@ public function testOtherAggregates(): void
                 'items_code' => ['aggregate' => 'count', 'field' => 'code', 'type' => 'integer'], // counts only not-null values
                 'items_star' => ['aggregate' => 'count', 'type' => 'integer'], // no field set, counts all rows with count(*)
                 'items_c:' => ['concat' => '::', 'field' => 'name'],
-                'items_c-' => ['aggregate' => $i->getPersistence()->dsql()->groupConcat($i->expr('[name]'), '-')],
-                'len' => ['aggregate' => $i->expr('SUM(' . $makeLengthSqlFx('[name]') . ')'), 'type' => 'integer'],
+                'items_c-' => ['aggregate' => $i->getPersistence()->dsql()->groupConcat($l->getReference('Items')->createTheirModel()->expr('[name]'), '-')],
```

Comment:
Ideally we should preserve the model instance in Expression and assert later that the instance is present in the whole SQL (owner model) - however this will fail after the owner model is cloned...